CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/PluginParametersDialog.java

```diff
@@ -74,13 +75,14 @@ public PluginParametersDialog(final Window owner, final String title, final Plug
 
         final BorderPane root = new BorderPane();
         root.setPadding(new Insets(10));
-        root.setStyle("-fx-background-color: #DDDDDD;");
         final Scene scene = new Scene(root);
+        scene.getStylesheets().addAll(JavafxStyleManager.getMainStyleSheet());
         setScene(scene);
 
-        final PluginParametersPane parametersPane = PluginParametersPane.buildPane(parameters, null);
-        root.setCenter(parametersPane);
-
+        if (parameters != null) {
+            final PluginParametersPane parametersPane = PluginParametersPane.buildPane(parameters, null);
+            root.setCenter(parametersPane);
+        }
```

Comment:
noting that this is for a `PluginParametersDialog`, I wonder whether it would actually make more sense to throw an exception if parameters is null (given there doesn't seem like much use without any). Maybe assert that `parameters != null` rather than an if check